### PR TITLE
fix(cli): allow running quick maintenance with epoch manager

### PR DESCRIPTION
**Pull request by julio-lopez**
- Fixes: #2775
- Fixes: #3600
- #4185

### CODE REVIEW

File: cli/command_logs_test.go [hunk 1]
```diff
@@ -90,14 +90,24 @@ func TestLogsMaintenance(t *testing.T) {
...
 
 	e.RunAndExpectSuccess(t, "maintenance", "set", "--max-retained-log-age=1ms")
 	e.RunAndVerifyOutputLineCount(t, 4, "logs", "list")
 
-	e.RunAndExpectSuccess(t, "maintenance", "run")
+	e.RunAndExpectSuccess(t, "maintenance", "run", "--full")
 	e.RunAndVerifyOutputLineCount(t, 1, "logs", "list")
+}
+
+func TestLogsMaintenanceSet(t *testing.T) {
```
Comment (julio-lopez) [inline cli/command_logs_test.go:103]: This change is unrelated to the primary purpose of the PR.
It simply splits this check to its own test.

**Review by jkowalski — APPROVED** [review]
